### ISSUE TITLE
scheduler callback: Invalid argument(s)

### ISSUE THREAD

**Issue by hs-dev1** [kay-af/slide-action#2]
here I got the error with flutter web. => when I pass color or icon as a return to Builders I get the following issue:

The following ArgumentError was thrown during a scheduler callback:
Invalid argument(s): Invalid parameters for blend mode ColorFilter

please let me know any way to solve it.

**Comment by kay-af** [comment]
Can you post some code so I could reproduce the error?

**Comment by kay-af** [comment]
Hi @mhussnainshabbir. Thank you for pointing out the error. It seems like the color blend modes support is currently broken on flutter web. Check [flutter issue 61780](https://github.com/flutter/flutter/issues/61780).  

Although the issue has been closed, I am still facing the error even in the latest version of Flutter on flutter web.  

I have updated the package to version 0.0.2 where this error does not occur anymore for the web. However, the *disabledColorTint* property is currently ignored on the web. When the widget is disabled, the opacity is reduced instead for the web. For mobile platforms, the property still works as expected.  

I am closing this issue now.
